### ISSUE TITLE
Toolbar buttons mask Vimium/Pentadactyl accessibility link tooltips

### ISSUE THREAD

**Issue by nickstenning** [hypothesis/client#242]
_Original issue: hypothesis/h#2759, reported by @holtzermann17._

> I use Pentadactyl (http://5digits.org/pentadactyl/), which allows me to follow URLs by number using the keyboard. Specifically, pressing <kbd>l</kbd> followed by <kbd>`<number>`</kbd> first decorates the links with numbers and then visits link numbered `<number>`.  But the Hypothes.is toolbar manages to have a z-index that keeps it on top of the numbers.
>
>![zindex](https://cloud.githubusercontent.com/assets/179958/11503302/c60f86b0-9837-11e5-97d5-8d3b3443c6a9.jpg)

### Steps to reproduce
1. Download and install [Pentadactyl (FF)](http://5digits.org/pentadactyl/) or [Vimium (Chrome)](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb) which offers similar functionality.
2. Visit a page with Hypothesis embedded, such as [this one](https://internethealthreport.org/v01/web-literacy/).
3. Type <kbd>l</kbd> (Pentadactyl) or <kbd>f</kbd> (Vimium) to activate the link-hinting mode.

### Expected behaviour
The link hint popovers for the toolbar buttons should be visible as they are for other links on the page.

### Actual behaviour
The toolbar buttons mask the link hints as visible in the screenshot above and here (for Vimium): 
![screen shot 2017-02-21 at 11 55 06](https://cloud.githubusercontent.com/assets/3602/23162033/b759ba36-f82c-11e6-9a18-10ec2d9f12d6.png)

### Browser/system information
Tested and verified with Chrome 58.0.3013.3 (Official Build) dev (64-bit) and Vimium 1.57.

**Comment by holtzermann17** [comment]
![z](https://cloud.githubusercontent.com/assets/179958/23864766/704aa63e-080b-11e7-814c-809116e4090c.jpg)
